tests/src/main/java/ma/glasnost/orika/test/generator/TestCompilerStrategyWritingFiles.java

```diff
@@ -87,7 +87,8 @@ public static void tearDown() {
     	
     	} finally {
     		// Note: this is coming from hibernate jar which is included only for test
-    		IOUtils.deleteRecursive(generatedSrc.getParentFile().getAbsolutePath(), true);
+    		//TODO: is this still needed? The function doesn't exist anymore
+    		//IOUtils.deleteRecursive(generatedSrc.getParentFile().getAbsolutePath(), true);
```

Comment:
Removing this doesn't seem to change anything.  
Maybe it was an issue only with the old hibernate version.